packages/css/src/card/card.css

```diff
@@ -8,12 +8,13 @@
   border-bottom: initial;
   text-decoration: initial;
   background: var(--hds-colors-lighter);
-  border-radius: var(--hds-border-radius);
+  border-radius: var(--hds-border-radius-16);
   transition: box-shadow var(--hds-micro-animation-duration-normal)
     var(--hds-micro-animation-easing-normal);
   display: flex;
   flex-direction: column;
   color: var(--hds-ui-colors-black);
+  padding: var(--hds-spacing-20-24);
```

Comment:
Looking at Figma, yes. https://www.figma.com/design/64ieYInJRps5diLI1qgJJk/Hedwig-Design-System?node-id=16258-6187&t=iEBJVXSk5QDaSb4x-1

ORF team also uses card on this page: https://sending.posten.no/landpostbud/
It will then look like this

| Header | Header |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/3978e3fa-9b0f-4940-9a12-7215328b6de1) | ![image](https://github.com/user-attachments/assets/8b9bce0a-5f04-4279-a3ab-c30c3d8e6d0c) |

Reply:
I see that for most cards, the padding has been moved from around the body to around the entire card. Except horizontal, which should have padding for both. Going over it again.